pages/locationdetail/[id].js

```diff
@@ -1,28 +1,37 @@
-import LocationCard from "@/components/LocationCard";
+import Card from "@/components/Card";
 import { useRouter } from "next/router";
-import ColonyList from "@/components/ColonyList";
-import StyledAddButton from "@/components/StyledAddButton";
 import Link from "next/link";
-import styled from "styled-components";
-
-export const StyledLink = styled(Link)`
-  position: fixed;
-  bottom: 100px;
-  left: 10px;
-`;
+import StyledAddLink from "@/components/StyledLinks";
+import { StyledList } from "@/components/StyledList";
+import { StyledBackLink } from "@/components/StyledLinks";
 
 export default function LocationDetailsPage({ locations, colonies }) {
   const router = useRouter();
-  const id = router.query.id;
+  const { id } = router.query;
+  const locationId = id;
```

Comment:
Sorry. I left the comment at the wrong place. I meant this place where you can delete the `locationId`.